js/apps/account-ui/src/personal-info/FormField.tsx

```diff
@@ -80,7 +80,10 @@ export const FormField = ({ attribute }: FormFieldProps) => {
       ) : (
         <KeycloakTextInput
           id={attribute.name}
-          {...register(fieldName(attribute.name))}
+          isDisabled={attribute.readOnly}
+          {...register(fieldName(attribute.name), {
+            required: { value: attribute.required, message: t("required") },
+          })}
```

Comment:
This seems unrelated to the issue the PR is trying to fix, or am I missing something here?